### ISSUE TITLE
Models should be aware of table prefixes

### ISSUE THREAD

**Issue by Ramir1** [cmgmyr/laravel-messenger#30]
```
    Column not found: 1054 Unknown column 'participants.last_read' in 'where clause' 
(SQL: select `l5_threads`.* from `l5_threads` inner join `l5_participants` on 
`l5_threads`.`id` = `l5_participants`.`thread_id` where `l5_threads`.`deleted_at` 
is null and `l5_participants`.`user_id` = 9 and `l5_participants`.`deleted_at` is null 
and (`l5_threads`.`updated_at` > participants.last_read 
or `l5_participants`.`last_read` is null) order by `updated_at` desc)
```
Don't used table prefix.
/cmgmyr/messenger/src/Cmgmyr/Messenger/Models/Thread.php:
-120 $query->where('threads.updated_at', '>', $this->getConnection()->raw('participants.last_read'))
+120 $query->where('threads.updated_at', '>', $this->getConnection()->raw(*$this->getConnection()->getTablePrefix().*'participants.last_read'))

**Comment by cmgmyr** [comment]
Thank you for reporting this. I'll take a look shortly

**Comment by cmgmyr** [comment]
This has been fixed in the latest release (2.2.1)
